tests/ambiguity_detection.rs

```diff
@@ -0,0 +1,78 @@
+//! A test to confirm that `bevy` doesn't have system order ambiguity with DefaultPlugins
+//! This is run in CI to ensure that this doesn't regress again.
+
+use bevy::{
+    ecs::schedule::{LogLevel, ScheduleBuildSettings, ScheduleLabel},
+    prelude::*,
+};
+//use bevy_render::{pipelined_rendering::RenderExtractApp, Render, RenderApp};
+
+/// FIXME: bevy should not have any ambiguities, but it takes time to clean these up,
+/// so we're juste ignoring those for now.
+fn get_ignored_ambiguous_systems() -> Vec<Box<dyn ScheduleLabel>> {
+    vec![
+        Box::new(First),
+        Box::new(PreUpdate),
+        Box::new(PostUpdate),
+        Box::new(Last),
+        Box::new(ExtractSchedule),
+        //Box::new(Render),
```

Comment:
This works as an example, but I'm not sure how to enable that for a test ? @alice-i-cecile